src/index.css

```diff
@@ -89,13 +98,18 @@ line-height: 24px;
   margin-top: 8px;
 }
 
-@media only screen
-     and (width: 430px) 
-     and (device-height: 932px)
-     and (-webkit-device-pixel-ratio: 3) {
+@media (min-width: 431px) and (max-width: 834px){
+  h1{
+    margin-left: 24px; 
+    font-size: 48px;
+    line-height: 56px;
+    }
+}
 
-      h1{
-        font-size: x-large;
-        color: red;
-      }
+@media (min-width: 835px){
```

Comment:
Btw: I could've been more careful when choosing the frames for the different viewport sizes in Figma 😅 
835px is a very uncommon breakpoint.
You can see here which breakpoints would've been more common: https://www.browserstack.com/guide/responsive-design-breakpoints

<img src="https://browserstack.wpenginepowered.com/wp-content/uploads/2024/08/Common-Breakpoints-for-Responsive-Design.png" width="598" />

Reply:
OK, I'll keep it in mind & maybe add to my learnings, but for now I'll keep it as figma designs (unless you object haha)